adagrams/game.py

```diff
@@ -1,13 +1,132 @@
 from random import randint
 
+LETTER_POOL = {
+    'A': 9, 
+    'B': 2, 
+    'C': 2, 
+    'D': 4, 
+    'E': 12, 
+    'F': 2, 
+    'G': 3, 
+    'H': 2, 
+    'I': 9, 
+    'J': 1, 
+    'K': 1, 
+    'L': 4, 
+    'M': 2, 
+    'N': 6, 
+    'O': 8, 
+    'P': 2, 
+    'Q': 1, 
+    'R': 6, 
+    'S': 4, 
+    'T': 6, 
+    'U': 4, 
+    'V': 2, 
+    'W': 2, 
+    'X': 1, 
+    'Y': 2, 
+    'Z': 1
+}
+
+HAND_SIZE = 10
+PILE_SIZE = len(LETTER_POOL)
+
+def make_new_draw_pile():
+    draw_pile = []
+    for letter, count in LETTER_POOL.items():
+        [draw_pile.append(letter)] * count
+    return draw_pile
```

Comment:
👍 Great idea to make a function that's responsible for building the pile from our tile counts.

👀 However, while I can see that's the intent of this code, the logic isn't carrying out that intention.

The line
```py
        [draw_pile.append(letter)] * count
```
doesn't append `count` copies of a value to the list. If it did, the final value of `draw_pile` would be a list with 98 tiles in it (which is what we'd want to see). But if you inspect the return value, you'll see something else.

This logic should be revised to carry out the pile-building intent. But also be sure that you understand what the current code is doing and why.